src/commands/distribute/release.ts

```diff
@@ -144,83 +120,34 @@ export default class ReleaseBinaryCommand extends AppCommand {
         throw statusCode;
       }
     } catch (error) {
-      if (error === 404) {
-        throw failure(ErrorCodes.InvalidParameter, `distribution group ${this.distributionGroup} was not found`);
-      } else {
-        debug(`Failed to get users of distribution group ${this.distributionGroup}, returning null - ${inspect(error)}`);
-        return null;
-      }
+      throw failure(ErrorCodes.InvalidParameter, `distribution group ${this.distributionGroup} was not found`);
```

Comment:
So if the server returns 500 we show a message "distribution group <name> was not found"? That will be incorrect and misleading. 
We should show a generic error message with a context ID (If context ID is shown when running with `--debug` in `client.distributionGroups.listUsers` then having a context ID printed here is just nice to have).